Modules/_collectionsmodule.c

```diff
@@ -1150,11 +1150,11 @@ deque_insert(dequeobject *deque, PyObject *const *args, Py_ssize_t nargs)
 }
 
 PyDoc_STRVAR(insert_doc,
-"insert(i, x)\n\n"
+"insert(i, x)\n"
 "Insert *x* into the deque at position *i*.");
 
 PyDoc_STRVAR(remove_doc,
-"remove(x)\n\n"
+"remove(x)\n"
 "Remove the first occurrence of *x*."
```

Comment:
Here's another regression from #100990: there's no space between the doc and the following sentence.